popout.js

```diff
@@ -143,20 +145,21 @@ class PopoutModule {
 				// Add delay before rendering in case some things aren't done initializing, like sheet templates
 				// which get loaded asynchronously.
 				Hooks.on('ready', () => setTimeout(() => PopoutModule.renderPopout(${sheet}), 1000));
-		  	window.dispatchEvent(new Event('load'))
-		  </script>`))
+		  	window.dispatchEvent(new Event('load'));
+		  </script>`));
 		// Open new window and write the new html document into it
 		// We need to open it to the same url because some images use relative paths
 		let windowFeatures = undefined;
 		if (game.settings.get("popout", "useWindows"))
-			windowFeatures = 'toolbar=0,location=0,menubar=0,titlebar=0,scrollbars=1';
-		let win = window.open(window.location.toString(), '_blank', windowFeatures)
+			windowFeatures = 'toolbar=0, location=0, menubar=0, titlebar=0, scrollbars=1, width=800, height=900';

```

Comment:
I assume this would affect popout windows rather than tabs, right? When I tested, if I remember correctly, it would set the size to the same size as the FVTT window itself, why would forcing it to a fixed size be better here ?

Reply:
It always pops out as a window when I use it in chrome, regardless of whether I have useWindows checked or not in the module.  The size changes are just to pop it out as a smaller window for convenience.

Reply:
Someone else on a laptop might have it pop out bigger. It makes more sense to keep it set to whatever the user had the window at rather a fixed size for convenience to you specifically.
An alternative would be to get the width and height of the sheet itself from foundry and pop the new window at that exact size. Would need to be careful about window decoration/menus/padding/etc.. to make sure the sheet itself remains at the same size as it was in within foundry. I think that'd be a good solution actually.